manim/mobject/types/vectorized_mobject.py

```diff
@@ -1018,7 +1018,7 @@ def gen_subpaths_from_points_2d(self, points):
     def get_subpaths(self) -> typing.Tuple:
         """Returns subpaths formed by the curves of the VMobject.
 
-        We define a subpath between two curve if one of their extreminities are coincidents.
+        We define a subpath between two curve if one of their extremities are coincident.
```

Comment:
I think this should be reformulated, because it's terribly unclear. 
Why not 
`Two curves form a subpath iff one of their extremities are coincident`